fretboard-scheduler-jobscheduler/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="mozilla.components.service.fretboard.scheduler.jobscheduler">
+
+    <application>
+        <service android:name=".SyncJob"
+            android:permission="android.permission.BIND_JOB_SERVICE" />
+    </application>
```

Comment:
With the proposed change above this would be up to the app to add (and something we would need to document in that module's README)